docs/installation.md

```diff
@@ -59,8 +59,8 @@ Or install a specific version with:
 ```
 
 > [!WARNING]
-> Linux users need to have `inotify-tools` installed for the VSCode filewatcher to work properly.
-> This impacts how the extension presents file-specific data.
+>
+> [VSCode uses inotify](https://github.com/microsoft/vscode/wiki/File-Watcher-Issues) on Linux installations. If file changes aren't propagating in the Publisher extension, ensure you have `inotify` installed.
```

Comment:
> If file changes aren't propagating

Nitpicky: This wording made me think of seeing the file changes line by line, but not the type of updates we do. I think something more vague like "If file changes aren't updating the Publisher extension,...".

Feel free to ignore this bit of feedback.

Reply:
Good point, I'll update it.